src/js/EditorBlock.js

```diff
@@ -84,7 +99,47 @@ export default class EditorBlock extends Component {
             )
         }
 
-        const keyInput = (
+        const userAPIKeyMessage = this.state.isSavedKey ? (
+            <div>
+                <p style={{textAlign: 'center'}}>
```

Comment:
This could be wrapped in a conditional that checks if `! PantheonGoogleMapsData.userCanManageOptions` to avoid the code duplication